data/src/main/java/org/apache/iceberg/data/DeleteFilter.java

```diff
@@ -192,13 +196,10 @@ protected void markRowDeleted(T item) {
   }
 
   public Predicate<T> eqDeletedRowFilter() {
-    if (eqDeleteRows == null) {
```

Comment:
Do we need to change this?

Reply:
Yes,The function to avoid initialize 'delete row' of eqDeleteRows is the same as eqDeletePredicates.

Reply:
I don't think this needs to change. The unified predicate and the list of predicates can coexist.

Reply:
I get you